src/uvwasi.c

```diff
@@ -2557,6 +2557,16 @@ uvwasi_errno_t uvwasi_sock_shutdown(uvwasi_t* uvwasi,
   return UVWASI_ENOTSUP;
 }
 
+uvwasi_errno_t uvwasi_sock_accept(uvwasi_t* uvwasi,
+                                  uvwasi_fd_t sock,
+                                  uvwasi_fdflags_t flags,
+                                  uvwasi_fd_t* fd) {
+  /* TODO(mhdawson): Waiting to implement, pending
+                    https://github.com/WebAssembly/WASI/issues/4 */
```

Comment:
Having a TODO is good, but I don't think this specific text applies in this case.

Reply:
@cjihrig updated.